From 2240d39532cce0cc140134457cb46e1d271be5d4 Mon Sep 17 00:00:00 2001
From: Paul Keith <javelinanddart@gmail.com>
Date: Fri, 19 Jan 2018 04:52:42 +0100
Subject: [PATCH 1/2] msm8974: Enable full dex preopt

* So, uh, it turns out we need to disable booting with all
  but 1 CPU core to fix CDMA RIL on klte (and hlte?), so
  let's enable full dex preopt so that we don't need to
  dex opt after system updates

Change-Id: I6fe57f9dcd1da8245eebd51d61055d6de7ffe6cd
---
 BoardConfigCommon.mk | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/BoardConfigCommon.mk b/BoardConfigCommon.mk
index de7d1ef..45f0255 100644
--- a/BoardConfigCommon.mk
+++ b/BoardConfigCommon.mk
@@ -41,10 +41,7 @@ WITH_LINEAGE_CHARGER := false
 # Dexpreopt
 ifeq ($(HOST_OS),linux)
   ifneq ($(TARGET_BUILD_VARIANT),eng)
-    ifeq ($(WITH_DEXPREOPT),)
-      WITH_DEXPREOPT := true
-      WITH_DEXPREOPT_BOOT_IMG_AND_SYSTEM_SERVER_ONLY := true
-    endif
+    WITH_DEXPREOPT := true
   endif
 endif
 
-- 
2.7.4

